examples/generate-python-complete-models/__snapshots__/index.spec.ts.snap

```diff
@@ -5,24 +5,24 @@ Array [
   "
 
 class ObjProperty: 
-  def __init__(self, input):
-    if hasattr(input, 'number'):
-    	self._number = input.number
-    if hasattr(input, 'additionalProperties'):
-    	self._additionalProperties = input.additionalProperties
+  def __init__(self, input: dict):
```

Comment:
Ideally, for backwards compatibility we should use `Dict` instead of `dict` here, and same goes for `List` vs `list` in type annotations. Both of those need `from typing import Dict` or `List`

Reply:
Do you think we should include it in this PR, or another? 🤔

Reply:
Changed it :v: